static/js/place/place_template.tsx

```diff
@@ -568,9 +579,7 @@ class ChartBlock extends Component<ChartBlockPropType, unknown> {
     conf.title = conf.title + " in " + this.props.placeName;
     result.push(conf);
 
-    conf = { ...config };
-    conf.chartType = chartTypeEnum.GROUP_BAR;
-    conf.axis = "PLACE";
+    conf = this.copyAndUpdateConfig(config);
```

Comment:
i missed this in my last PR - could you also use displayPlaceType for the else clause (line 588)

Reply:
it should be okay there now